src/Docker/Configuration/Container.php

```diff
@@ -77,6 +77,13 @@ public function getConfigTreeBuilder()
         $outputTable = $output
             ->children()
                 ->scalarNode('default_bucket')->end()
+                ->enumNode('data_types')
+                    ->values(['authoritative', 'no-types', 'null'])
```

Comment:
Miro to tady pojmenoval nakonec trošku jinak https://github.com/keboola/developer-portal/pull/451/files#diff-dfcd8afcb372a75a1eb1bdc0af57d36a5e0a727dbca99350779abb21333a298fR202

upravto pak prosímtě i v dokumentu v confluence

Reply:
I ty hodnoty :)